django_rq/admin.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib import admin
-from django_rq import settings
+
+from . import settings
```

Comment:
There was a mix of relative and absolute imports within `django_rq` itself - I've made all internal package import relative, although I have left imports from `django_rq.tests` to `django_rq.*` as absolute?